edalize/utils.py

```diff
@@ -17,6 +17,9 @@ def add(self, command, targets, depends, order_only_deps=[]):
     def add_var(self, var):
         self.vars.append(var)
 
+    def add_env_var(self, key, value):
```

Comment:
It would be great if we can avoid this since it's not portable to e.g. windows or csh shells. I think also ninja (which we probably want to use instead of make eventually) doesn't support setting variables this way

Reply:
I took a quick glance at the env vars used in the flow. Are any of them used internally in the tools, or just as a shorthand? e.g. the LOOKAHEAD variable is used later on in two places. If it's _not_ used internally in the flows then I think it's better that Edalize just expands them. Then we can take a look at what to do with the ones used internally in the tools. My opinion is that using env vars it's not a super nice API for a tool, and perhaps there are other ways to pass this information

Reply:
Ultimately, we would want to simplify/remove the scripts or have them generated by edalize, so using environmental variables is not the best way, but until that, this is the best solution I came up with.

Reply:
@olofk What do you think of something like this? https://github.com/byuccl/edalize/commit/b8643720442035e6f29969dc66c59643b98df96c